revista-medica-del-instituto-mexicano-del-seguro-social.csl

```diff
@@ -0,0 +1,430 @@
+                <if variable="container-title-short">
+                  <text variable="container-title-short" strip-periods="true"/>
+                </if>
+                <!-- 2) Si no, intentar la forma corta -->
+                <else>
+                  <text variable="container-title" form="short" strip-periods="true"/>
+                </else>
+              </choose>
+            </if>
+            <!-- Para el resto: forma larga -->
+            <else>
+              <text variable="container-title" strip-periods="true"/>
+            </else>
+          </choose>
+          <!-- Añadir [Internet] solo si hay URL -->
+          <choose>
+            <if variable="URL">
+              <text term="internet" prefix="[" suffix="]" text-case="capitalize-first"/>
+            </if>
+          </choose>
+        </group>
+        <!-- Edición si aplica -->
+        <text macro="edition" prefix=" "/>
+      </if>
+      <!-- Legislación -->
+      <else-if type="bill legislation" match="any">
+        <group delimiter=", " suffix=".">
+          <group delimiter=". ">
+            <text variable="container-title"/>
+            <group delimiter=" ">
+              <text term="section" form="short" text-case="capitalize-first"/>
+              <text variable="section"/>
+            </group>
+          </group>
+          <text variable="number"/>
+        </group>
+      </else-if>
+      <!-- Discursos / presentaciones -->
+      <else-if type="speech">
+        <group delimiter=": " suffix=";">
+          <group delimiter=" ">
+            <text variable="genre" text-case="capitalize-first"/>
+            <text term="presented at"/>
+          </group>
+          <text variable="event"/>
+        </group>
+      </else-if>
+      <!-- Otros tipos -->
+      <else>
+        <group delimiter=", " suffix=".">
+          <choose>
+            <if variable="collection-title" match="none">
+              <group delimiter=" ">
+                <label variable="volume" form="short" text-case="capitalize-first"/>
+                <text variable="volume"/>
+              </group>
+            </if>
+          </choose>
+          <text variable="container-title"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="title">
+    <!-- Título principal del documento -->
+    <text variable="title"/>
+    <!-- Si el tipo NO es artículo, capítulo o reseña, añade [internet] y edición -->
+    <choose>
+      <if type="article-journal article-magazine chapter paper-conference article-newspaper review review-book entry-dictionary entry-encyclopedia" match="none">
+        <!-- Si tiene URL, muestra [internet] en minúscula -->
+        <choose>
+          <if variable="URL">
+            <text term="internet" prefix=" [" suffix="]" text-case="capitalize-first"/>
+          </if>
+        </choose>
+      </if>
+    </choose>
+    <!-- Si el documento es una tesis, muestra el tipo de tesis (p. ej. [Tesis de maestría]) -->
+    <choose>
```

Comment:
Instead of having 2 conditionals you could add the "thesis" one via an else-if into the first conditional.